### PR TITLE
Add mindone transformers model tests

### PR DESCRIPTION
**Pull request by CaitinZhao**
# What does this PR do?

1. Add transformers model tests in mindone

- refer to huggingface transformers tests
- support dtype in [fp32， fp16， bf16]
- support ms mode in [0, 1]

2. Adjusting the directory structure
 tests
     diffusers_tests
     transformers_tests
     other_tests

3. fix some bugs

Fixes # (issue)

Adds # (feature)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: tests/diffusers_tests/test_t5_film_transformer.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 import mindspore as ms
 
-from .modeling_test_utils import compute_diffs, generalized_parse_args, get_modules
+from ..modeling_test_utils import compute_diffs, generalized_parse_args, get_modules
 
 THRESHOLD_FP16 = 1e-2
```
Comment (SamitHuang) [inline tests/diffusers_tests/test_t5_film_transformer.py:9]: why 1e-2

File: tests/transformers_tests/models/clip/test_modeling_clip.py [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+            "text_embeds": 2,
+            "image_embeds": 3,
+        },
+    ],
+]
+
+
+@pytest.mark.parametrize(
+    "name,pt_module,ms_module,init_args,init_kwargs,inputs_args,inputs_kwargs,outputs_map,dtype,mode",
+    [case + context for case in CLIP_CASES for context in [["fp16", 0], ["fp16", 1], ["fp32", 0], ["fp32", 1]]],
```
Comment (SamitHuang) [inline tests/transformers_tests/models/clip/test_modeling_clip.py:280]: what about bf16?

File: tests/diffusers_tests/test_t5_film_transformer.py (hunk 1 above)
Comment (CaitinZhao) [reply tests/diffusers_tests/test_t5_film_transformer.py:9]: This is the unified baseline under diffuser tests.

File: tests/transformers_tests/models/clip/test_modeling_clip.py (hunk 2 above)
Comment (CaitinZhao) [reply tests/transformers_tests/models/clip/test_modeling_clip.py]: add bf16 tests

**Review by SamitHuang — APPROVED** [review]
LGTM

**Review by vigo999 — APPROVED** [review]

**Pull request merged**